osf/management/commands/migrate_pagecounter_data.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from framework import sentry
+
+logger = logging.getLogger(__name__)
+
+
+LIMIT_CLAUSE = ' LIMIT %s);'
+NO_LIMIT_CLAUSE = ');'
+
+REVERSE_SQL_BASE = '''
+UPDATE osf_pagecounter PC
+SET
+    resource_id = NULL,
+    file_id = NULL,
+    version = NULL
```

Comment:
Might as well clear `action` as well?

Reply:
_Apparently_ `action` has a non-null constraint on it.

Reply:
I think I should take that off in my first PR, apparently I added a default value of 'download', at first, in the schema migration. to have the non-null constraint on it, for some reason. I think we should have non-null constraints in the end, but don't need it at first.